documentation/src/main/asciidoc/topics/ref_discovery_jdbc.adoc

```diff
@@ -1,7 +1,12 @@
+[id='discovery_jdbcping-{context}']
 = JDBC_PING
-`JDBC_PING` uses JDBC connections to shared databases, such as Amazon RDS on
-EC2, to store information about {brandname} nodes.
+JDBC_PING uses shared databases to store information about {brandname} clusters.
+This protocol supports any database that can use a JDBC connection.
```

Comment:
worth mention (as a warning/note) that the user needs to add the driver jar (H2, MySQL, whatever he is using)